src/styles/variables.css

```diff
@@ -0,0 +1,6 @@
+:root {
+  --main-color-white: #fff;
```

Comment:
I don't sure you need to create a variable for so simple color

Reply:
Done!

Reply:
There is no difference in how simple this color is, the question is how many times you use it.

At now, you use this color 3 times in the code (or maybe even more), so if you will decide to change this color to #ff9, you will need to do it in 3 different places, instead of 1.

So variable was a good idea🙃